src/status_im/subs/wallet/screens/input_amount.cljs

```diff
@@ -0,0 +1,360 @@
+        zero-owned-eth? (money/equal-to (:total-balance owned-eth-token) 0)
+        exceeded-input? (if eth-selected?
+                          upper-limit-equals-input-value?
+                          zero-owned-eth?)]
+    (and (or no-routes-found? limit-exceeded?)
+         (seq sender-network-values)
+         (or exceeded-input? (not enough-assets?)))))
+
+(rf/reg-sub :send-input-amount-screen/not-enough-asset?
+ :<- [:wallet/wallet-send-sender-network-values]
+ :<- [:wallet/wallet-send-enough-assets?]
+ :<- [:wallet/wallet-send-token]
+ :<- [:send-input-amount-screen/upper-limit-equals-input-value?]
+ :<- [:send-input-amount-screen/no-routes-found?]
+ :<- [:send-input-amount-screen/upper-limit-exceeded?]
+ :<- [:send-input-amount-screen/enabled-from-chain-ids]
+ (fn [[sender-network-values
+       enough-assets?
+       {token-symbol :symbol}
+       upper-limit-equals-input-value?
+       no-routes-found?
+       limit-exceeded?
+       enabled-from-chain-ids]]
+   (let [owned-eth-token (rf/sub [:wallet/token-by-symbol
+                                  (string/upper-case constants/mainnet-short-name)
+                                  enabled-from-chain-ids])]
+     (insufficient-asset-amount?
+      {:enough-assets?                  enough-assets?
+       :token-symbol                    token-symbol
+       :owned-eth-token                 owned-eth-token
+       :upper-limit-equals-input-value? upper-limit-equals-input-value?
+       :no-routes-found?                no-routes-found?
+       :limit-exceeded?                 limit-exceeded?
+       :sender-network-values           sender-network-values}))))
+
+(rf/reg-sub :send-input-amount-screen/should-try-again?
+ :<- [:send-input-amount-screen/upper-limit-exceeded?]
+ :<- [:send-input-amount-screen/no-routes-found?]
+ :<- [:send-input-amount-screen/not-enough-asset?]
+ (fn [[limit-exceeded?
+       no-routes-found?
+       not-enough-asset?]]
+   (and (not limit-exceeded?)
+        no-routes-found?
+        (not not-enough-asset?))))
+
+(rf/reg-sub :send-input-amount-screen/show-no-routes?
+ :<- [:send-input-amount-screen/upper-limit-exceeded?]
+ :<- [:send-input-amount-screen/no-routes-found?]
+ :<- [:send-input-amount-screen/not-enough-asset?]
+ :<- [:wallet/wallet-send-sender-network-values]
+ (fn [[limit-exceeded?
+       no-routes-found?
+       not-enough-asset?
+       sender-network-values]]
+   (and (or no-routes-found? limit-exceeded?)
+        (not-empty sender-network-values)
+        (not not-enough-asset?))))
+
+(rf/reg-sub :send-input-amount-screen/confirm-disabled?
+ :<- [:wallet/wallet-send-route]
+ :<- [:send-input-amount-screen/valid-input?]
+ (fn [[route
+       valid-input?]]
+   (or (nil? route)
+       (empty? route)
+       (not valid-input?))))
+
+(defn- get-fee-formatted
+  [route]
+  (when-let [native-currency-symbol (-> route first :from :native-currency-symbol)]
+    (rf/sub [:wallet/wallet-send-fee-fiat-formatted native-currency-symbol])))
```

Comment:
Same as another comment, `rf/sub` can't be used inside sub handlers. I think there are other instances of this problem.